include/drivers/lora.h

```diff
@@ -132,6 +143,27 @@ static inline int lora_send(const struct device *dev,
 	return api->send(dev, data, data_len);
 }
 
+/**
+ * @brief Asynchronously send data over LoRa
+ *
```

Comment:
Please add a note as like `lora_send` just to align.

Reply:
done